serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -9,12 +9,15 @@
 # or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS"
 # BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied. See the License for
 # the specific language governing permissions and limitations under the License.
-ARG version=12.2.2-runtime-ubuntu22.04
+ARG version=12.2.2-cudnn8-runtime-ubuntu22.04
```

Comment:
does it have tensorrt pre-installed?

Reply:
no

Reply:
curious why are we switching from runtime image?

Reply:
this is a clean install and save 2 GB of memory. Now the entire container takes 11GB uncompressed

Reply:
cudnn image takes less space than runtime image?

Reply:
takes more, but we anyway install everything manually, so it is the same

Reply:
during installation of tensorrt, we install cudnn8